#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_ignoretrunk.dpatch by  <crafterm@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Ignore TRUNK branch name patch

@DPATCH@

diff -urN cvsps-2.1.orig/cvsps.c cvsps-2.1/cvsps.c
--- cvsps-2.1.orig/cvsps.c	2005-05-25 22:39:40.000000000 -0500
+++ cvsps-2.1/cvsps.c	2005-06-19 23:07:20.000000000 -0500
@@ -2104,6 +2109,11 @@
     
     if (!get_branch_ext(rev, eot, &leaf))
     {
+	if (strcmp(tag, "TRUNK") == 0)
+	{
+	    debug(DEBUG_STATUS, "ignoring the TRUNK branch/tag");
+	    return;
+	}
 	debug(DEBUG_APPERROR, "malformed revision");
 	exit(1);
     }
